rules.rb

```diff
@@ -1 +1,132 @@
+### Berksfile ###
+#
 
+rule 'CB001', 'Missing Berksfile' do
+  tags %w(style berksfile brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'Berksfile'
+  end
+end
+# berkfile should source supermarket
+#  +source 'https://supermarket.chef.io'
+
+# cf CINK001
+rule 'CB002', 'Missing CHANGELOG in markdown format' do
+  tags %w(style changelog brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'CHANGELOG.md'
+  end
+end
+
+# rule 'CB003', 'CHANGELOG should follow keepachangelog.com format'
+
+rule 'CB004', 'Missing CODE_OF_CONDUCT in markdown format' do
+  tags %w(style code_of_conduct brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'CODE_OF_CONDUCT.md'
+  end
+end
+
+rule 'CB005', 'Missing CONTRIBUTING in markdown format' do
+  tags %w(style contributing brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'CONTRIBUTING.md'
+  end
+end
+
+### Gemfile ###
+
+# rule 'CB008', 'Gemfile should depend on berkshelf'
+# rule 'CB009', 'Gemfile should depend on chefspec'
+# rule 'CB010', 'Gemfile should depend on cookstyle'
+# rule 'CB011', 'Gemfile should depend on foodcritic'
+# rule 'CB012', 'Gemfile should depend on stove'
+# rule 'CB013', 'Gemfile should depend on test-kitchen'
+
+rule 'CB006', 'Missing LICENSE in markdown format' do
+  tags %w(style license brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'LICENSE.md'
+  end
+end
```

Comment:
I would say Markdown is standard except for licenses.

For example, searching all the files on GitHub:

- Percentage of use of the file *README.md* ([~ 85,000,000](https://github.com/search?utf8=%E2%9C%93&q=filename%3AREADME.md&type=Code&ref=searchresults) / [~ 104,000,000](https://github.com/search?utf8=%E2%9C%93&q=filename%3AREADME&type=Code&ref=searchresults)): **82%**
- Percentage of use of the file *LICENSE.md* ([~ 2,750,000](https://github.com/search?utf8=%E2%9C%93&q=filename%3ALICENSE.md&type=Code&ref=searchresults) / [~ 65,050,000](https://github.com/search?utf8=%E2%9C%93&q=filename%3ALICENSE&type=Code&ref=searchresults)): **4%**

And the same applies to chef community cookbooks. Indeed, both `chef generate repo` and `berks cookbook` commands create a *LICENSE* file without extension.

Anyway, I want to clarify that I'm not choosing one over the other. I'm proposing to **accept both**, not only the markdown version.